packages/scrabble-solver/package.json

```diff
@@ -20,10 +20,10 @@
   },
   "homepage": "https://github.com/kamilmielnik/scrabble-solver#readme",
   "scripts": {
-    "build": "NEXT_TELEMETRY_DISABLED=1 next build",
```

Comment:
Yeah I also had to remove this to be able to run it on my Windows computer. Sorry I forgot to mention it in the pull request message

Reply:
Windows build should be now fixed with https://github.com/kamilmielnik/scrabble-solver/pull/55, thanks for the report!